### PR TITLE
start of pipeline delayed by 5 min to avoid the same suffix as master

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: job-dsls/jobs/kie_dailyBuild_pipeline.groovy [hunk 1]
```diff
@@ -139,7 +139,7 @@ pipelineJob("${folderPath}/kieAllBuildPipeline-${kieMainBranch}") {
     }
 
     triggers {
-        cron("H 20 * * *")
+        cron("H 20 5 * *")
```
Comment (winklerm) [inline job-dsls/jobs/kie_dailyBuild_pipeline.groovy]: According to Jenkins help, the cron expression should be rather something such as "10 20 * * *" - the H stands for "hash", the format is "MINUTE HOUR DOM MONTH DOW".

Comment (winklerm) [reply job-dsls/jobs/kie_dailyBuild_pipeline.groovy]: The other branch daily build pipeline would need to have a different value then.

Comment (mbiarnes) [reply job-dsls/jobs/kie_dailyBuild_pipeline.groovy]: * 20 * * * and 05 20 * * * ?

Comment (winklerm) [reply job-dsls/jobs/kie_dailyBuild_pipeline.groovy]: Rather "0 20 * * *" and "05 20 * * *" - the first * would mean every minute.

Comment (mbiarnes) [reply job-dsls/jobs/kie_dailyBuild_pipeline.groovy]: changed this online to H 19 * * * (master) and left H 20 * * * (7.14.x) - will see tomorrow!

Comment (winklerm) [reply job-dsls/jobs/kie_dailyBuild_pipeline.groovy]: Sounds good to me!

**Pull request closed without merging**